### ISSUE TITLE
WCAG 4.1.2: Buttons must have discernible text (#printbtn)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2283]
#### Issue
`Buttons must have discernible text` ([button-name](https://dequeuniversity.com/rules/axe/3.2/button-name?application=msftAI))
#### Target application
[Documentation - Microsoft MakeCode](https://makecode.microbit.org/docs)
#### Element path
#printbtn
#### Snippet
`<button id="printbtn" class="circular ui icon right floated button hideprint" title="Print this page">
                <i class="icon print"></i>
            </button>`
#### How to fix
    Fix all of the following:
      Element is in tab order and does not have accessible text
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Comment by jwunderl** [comment]
This is fixed on locally served docs, but the change hasn't propagated to the docs page yet.

**Comment by jwunderl** [comment]
backend docs renderer was updated / deployed, so this is now fixed
